api/src/main/java/com/velocitypowered/api/proxy/player/TabList.java

```diff
@@ -43,6 +44,13 @@ public interface TabList {
    */
   void addEntry(TabListEntry entry);
 
+  /**
+   * Adds a {@link List} of {@link TabListEntry}'s to the {@link Player}'s tab list.
+   *
+   * @param entries to add to the tab list
+   */
+  void addEntries(List<TabListEntry> entries);
```

Comment:
It is not totally necessary to use a `List` here, you could simply use a `Collection` to allow any collection to be used. It would also be nice to add another method that accepts an array `addEntries(TabListEntry... entries)`

Reply:
In my experience, most methods such as this one even use Iterable instead of Collection or List - wouldn't that make more sense here?

Reply:
> In my experience, most methods such as this one even use Iterable instead of Collection or List - wouldn't that make more sense here?

Personally, I feel that `Collection` is the most appropriate and consistent for Velocity, since other methods also use Collection as a parameter or as the type of the object to return

[`Collection<TabListEntry>`](https://jd.papermc.io/velocity/3.0.0/com/velocitypowered/api/proxy/player/TabList.html#getEntries())
[`Collection<Player>`](https://jd.papermc.io/velocity/3.0.0/com/velocitypowered/api/proxy/ProxyServer.html#getAllPlayers())
[`Collection<RegisteredServer>`](https://jd.papermc.io/velocity/3.0.0/com/velocitypowered/api/proxy/ProxyServer.html#getAllServers())